pubspec.yaml

```diff
@@ -33,10 +33,10 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
-  ffigen: ^2.3.0
+  ffigen: ^6.0.0
   test: ^1.16.0
   yaml: ^3.0.0
-  pedantic: ^1.9.0
+  flutter_lints: ^2.0.0
```

Comment:
pedantic is deprecated.
https://pub.dev/packages/pedantic

However, I have not modified the code to match the flutter_lints because the scope of modification would be too large.